modules/oadp-using-legacy-aws-plugin.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-legacy-aws-plugin.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-using-legacy-aws-plugin_{context}"]
+= Using the legacy-aws Velero plugin in the DataProtectionApplication CR
+
+ In the following use case, you configure the `DataProtectionApplication` CR with the `legacy-aws` Velero plugin and then back up an application.
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have configured an {aws-short} S3-compatible object storage as a backup location.
+* You have an application with a database running in a separate namespace.
+
+.Procedure
+
+. Configure the `DataProtectionApplication` CR to use the `legacy-aws` Velero plugin as shown in the following example:
++
+.Example `DataProtectionApplication` CR
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: oadp-backup
+  namespace: openshift-adp
+spec:
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+        - legacy-aws # <1>
+        - openshift
+        - csi
+      defaultSnapshotMoveData: true 
+  backupLocations:
+    - velero:
+        config:
+          profile: "default"
+          region: noobaa
+          s3Url: https://s3.openshift-storage.svc 
+          s3ForcePathStyle: "true"
+          insecureSkipTLSVerify: "true"
+        provider: aws
+        default: true
+        credential:
+          key: cloud
+          name:  cloud-credentials
+        objectStorage:
+          bucket: <bucket_name> # <2>
+          prefix: oadp
+----
+<1>	Use the `legacy-aws` plugin.
```

Comment:
```suggestion
<1> Use the `legacy-aws` plugin.
```
Not sure if GitHub is rendering this oddly

Reply:
Yeah probably. I don't see extra space in the file.